src/test/fuzz/asmap_direct.cpp

```diff
@@ -14,7 +14,7 @@ void test_one_input(const std::vector<uint8_t>& buffer)
 {
     // Encoding: [asmap using 1 bit / byte] 0xFF [addr using 1 bit / byte]
     bool have_sep = false;
-    size_t sep_pos;
+    size_t sep_pos = 0;
```

Comment:
Don't know if it's guaranteed to have a value but in that case you could `size_t sep_pos = buffer.size();` and after the loop `assert(sep_pos < buffer.size())`.

Reply:
I took the recommended value from compiler warning message. But it seems that there is a much better solution to that proposed by @MarcoFalke